enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk_setup/hooks/use-sdk-iframe-embed-settings.ts

```diff
@@ -116,6 +119,8 @@ export const useSdkIframeEmbedSettings = ({
       setRawSettings({ ...settings, ...persistedSettings });
 
       setEmbedSettingsLoaded(true);
+
+      trackEmbedWizardOpened();
```

Comment:
This is what Timofey did in the useSdkIframeEmbedSettings hook in the main app - keeping it consistent.